lib/overcommit/hook/pre_commit/execute_permissions.rb

```diff
@@ -2,6 +2,16 @@ module Overcommit::Hook::PreCommit
   # Checks for files with execute permissions, which are usually not necessary
   # in source code files (and are typically caused by a misconfigured editor
   # assigning incorrect default permissions).
+  #
+  # Protip: if you have some files that you want to allow execute permissions
```

Comment:
Yes, it is. But, since people are filing issues asking this question, I think that is evidence enough that it could save people time to have a note here. It helps them apply the feature for this case, which I think is a win.